libbeat/processors/dns/dns_test.go

```diff
@@ -141,6 +141,7 @@ func TestDNSProcessorRunInParallel(t *testing.T) {
 
 	// Start several goroutines.
 	wg.Add(numGoroutines)
+	errs := make(chan error, numGoroutines*numEvents)
```

Comment:
One might want to use [`errgroup`](https://pkg.go.dev/golang.org/x/sync/errgroup), or [`Taskgroup`](https://pkg.go.dev/github.com/elastic/go-concert@v0.1.0/unison#TaskGroup).

Reply:
sure